src/Mesh.h

```diff
@@ -26,6 +32,8 @@ enum RxType {
 
   FISSION_RX,
 
+  NUFISSION_RX,
```

Comment:
The help information in RunTime.cpp(L436) should be changed accordingly.

Reply:
true. I'll need to start using that when I do parametric studies